syntax = "proto3";

package mqtt_comparison;

message SensorData {
  double timestamp = 1;
  string sensor_id = 2;
  double temperature = 3;
  double humidity = 4;
  double pressure = 5;
}
